cmd/app/app_test.go

```diff
@@ -40,42 +42,10 @@ func TestWorkspaceCommand(t *testing.T) {
 	listFunc = listPkgMock.List
 	listPkgMock.On("List").Return(nil)
 
-	err := cmd.Execute()
+	err := cmd.ExecuteContext(ctx)
```

Comment:
note: This is pretty much what ever change looks like 😆 `.Execute()` → `.ExecuteContext(ctx)`.